3_COLLABORATION/parcijelni_pull_requests/sandraBanka.py

```diff
@@ -0,0 +1,122 @@
+korisnici={
+    '1001':['Dora Pejacevic','1023456', 10000.00 ],
+    '1002':['Zinka Kunc','2345678', 2000.00],
+    '1003':['Slava Raškaj', '3456789', 5000.00],
+    '1004':['Dora Maar', '4567890', 2500.00]
+}
+
+administrator={ 
+    'anam':['Ana','Mihaljevic', 'aei123'],
+    'gork':['Goran', 'Kolar', 'bei456']}
+
+def logiranje(uneseni_username):
+    while uneseni_username not in administrator.keys():
```

Comment:
Nije netočno međutim, drugo pravilo Pythona kaže:
**Explicit is better than implicit.**  (https://peps.python.org/pep-0020/)

Ispravnije bi bilo reci, kada je u key onda to...
  while uneseni_username in administrator.keys():